src/main/docs/guide/configure.adoc

```diff
@@ -19,18 +19,7 @@ And then altering the configuration found in `grails-app/conf/application.yml`.
 include::{sourceDir}/grails-app/conf/application.yml[tags=dataSourceConfig,indent=0]
 ----
 
-To alter this for MySQL you should specify the driver class name, username and password:
```

Comment:
This change removes the line about altering for MySQL, need to combine the sentence above with the new content